File: Typescript/test/uniqueMorseRepresentations.test.ts
import { expect, test } from '@jest/globals';
import * as solution from '../src/uniqueMorseRepresentations';

test('Example 1', () => {
	let input = ["gin", "zen", "gig", "msg"];
	let expected = 2;
	expect(solution.uniqueMorseRepresentations(input)).toEqual(expected);
});

test('Example 2', () => {
	let input = ["a"];
	let expected = 1;
	expect(solution.uniqueMorseRepresentations(input)).toEqual(expected);
});

File: Dart/test/uniqueMorseRepresentations_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:leetcode/uniqueMorseRepresentations.dart';

void main() {
  test("test example 1", () {
    var input = ["gin", "zen", "gig", "msg"];
    var expected = 2;

    final sln = Solution();
    var output = sln.uniqueMorseRepresentations(input);
    expect(output, expected);
  });
  test("test example 2", () {
    var input = ["a"];
    var expected = 1;

    final sln = Solution();
    var output = sln.uniqueMorseRepresentations(input);
    expect(output, expected);
  });
}
